### ISSUE TITLE
Akka 2.6 in MQTT streaming fails Unidoc

### ISSUE THREAD

**Issue by ennru** [akka/alpakka#2203]
https://github.com/akka/alpakka/pull/2118 made MQTT Streaming use Akka 2.6.

This makes Unidoc fail as it tries to compile against Akka 2.5:

```
[error] /home/travis/build/akka/alpakka/mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/impl/ClientState.scala:510:18: value futureSource is not a member of object akka.stream.scaladsl.Source
[error] possible cause: maybe a semicolon is missing before `value futureSource'?
[error]                 .futureSource(reply.future)
[error]                  ^
[error] /home/travis/build/akka/alpakka/mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/impl/ClientState.scala:532:18: value futureSource is not a member of object akka.stream.scaladsl.Source
[error] possible cause: maybe a semicolon is missing before `value futureSource'?
[error]                 .futureSource(reply.future)
[error]                  ^

```

https://travis-ci.com/github/akka/alpakka/jobs/297873059#L578

Maybe using Akka 2.6 during Unidoc is a viable solution.

**Comment by ennru** [comment]
Using Akka 2.6 will not work either, as the Cassandra connector can't be compiled with it.

**Comment by raboof** [comment]
Ouch. For this particular one we could perhaps switch back to `fromFutureSource` (that one is deprecated but not removed in 2.6) - but then there will likely be other errors.

**Comment by raboof** [comment]
I'm confused though - this seems to work fine on master (https://travis-ci.com/github/akka/alpakka/jobs/298526606)?

**Comment by ennru** [comment]
You're right with #2206. The problem seems to be the exclusion which I didn't update in my recent PR.

**Comment by ennru** [comment]
Fixed with #2206
